src/modules/apiLikes.js

```diff
@@ -0,0 +1,9 @@
+const appId = 'CADKJQ2LFa6qH5nunRws';
+const urlLikesId = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${appId}/likes/`;
+// -- get likes from the API to use it in the next task
+const getLikes = async () => {
+  const data = await fetch(`${urlLikesId}`).then((response) => response.json());
```

Comment:
urlLinkesId is a variable, so don't need use ${}, just pass this variable as argument of fetch function like this `fetch(urlLinkesId)` .
According to JS best practice, I recommend you break your code into several like and use await instead of then. It would make your code more readable and clean.